docs/customer-api.md

```diff
@@ -397,4 +397,48 @@ PUT api/2/customers/6267d661e4b0420e3acced5e/streams/626fa59fe4b018f4ecdc092c/st
 
 Response example:
 
-[filename](_customer-api/_example-key-re-generate-StreamResponseDTO.md ':include')
\ No newline at end of file
+[filename](_customer-api/_example-key-re-generate-StreamResponseDTO.md ':include')
+
+## Pull restricted broadcast data
+
+Authorized customers are allowed to access Livery interactive restricted API calls. 
+Authorization will be done via an API key that is configured by a supervisor.
+
+The value of the added key must be included in header `x-livery-api-key`
+
+Unless an API key is added, the customer will not be able to retrieve broadcast data.
+
+On broadcast status change, it is possible to be notified of the changes. You have to add a value to `PUSH_URL` 
+server setting, representing your API endpoint which will receive the data with HTTP POST. 
+
+##### Note: Add your API endpoint URL to receive broadcast status changes. This is done via server settings update endpoint
+
+### Get broadcast data
```

Comment:
There are some details which should be described.
For example:
- the chat object only contains the 10 most recent messages.
- the leaderboard object only contains the first leaderboard page

Reply:
And maybe add some basic explanation:

This API call can be used to retrieve information about the broadcast that is currently live.
A use case is to poll this API to react on the broadcast being closed.